debug/debug.go

```diff
@@ -31,14 +31,16 @@ const (
 var (
 	// When this set to true, logger will print more events for debugging
 	Verbose = false
+	LoggerErr error
 )
 
 // This is a temporary solution for logging the shim-loggers-for-containerd package itself. We directly
 // send the events to system journal and they are identified by the package name. Since this process is
 // started by containerd, we can check the logs using `journalctl -u containerd.service`.
-func SendEventsToJournal(syslogIdentifier string, msg string, msgType journal.Priority) {
+func SendEventsToJournal(syslogIdentifier string, msg string, msgType journal.Priority, delaySeconds time.Duration) {
```

Comment:
Since we are multiplying this input param by `time.Second` anyway, I think it makes more sense for us to have this param as type `int`. Alternatively, we can let the caller specify a `time.Duration` param, which would allow us to pass this into `time.Sleep()` unmodified.

Reply:
If `delaySeconds` set to int, it will not be allowed to multiply `time.Second`. And I feel its kind of baggy to let the caller have input like `1 * time.Second` rather than `1` as this function is called everywhere. Wdyt?

Reply:
Ah okay I see, thanks for the explanation, I'm good with leaving it as-is.